src/main/java/org/wso2/carbon/module/swiftiso20022/utils/MT940ValidationUtils.java

```diff
@@ -186,32 +187,32 @@ public static ErrorModel validateReference(String referenceValue, String fieldNa
      * @param accIdentifier  Account identifier to be validated
      * @return    Whether account identifier is valid
      */
-    public static ErrorModel validateAccountIdentifier(String accIdentifier) {
+    public static ValidationResult validateAccountIdentifier(String accIdentifier) {
         String[] accIdentifierDetails = accIdentifier.split(ConnectorConstants.COLON);
         if (!isListLengthValid(accIdentifierDetails)) {
-            return new ErrorModel(ConnectorConstants.INVALID_REQUEST_PAYLOAD,
+            return new ValidationResult(ConnectorConstants.INVALID_REQUEST_PAYLOAD,
                     String.format(ConnectorConstants.ERROR_PARAMETER_INVALID,
-                            ConnectorConstants.ACC_IDENTIFICATION));
+                            MT940Constants.DN_ACC_IDENTIFICATION));
         }
         String account = accIdentifierDetails[2];
         if (StringUtils.isBlank(account)) {
-            return new ErrorModel(ConnectorConstants.INVALID_REQUEST_PAYLOAD,
+            return new ValidationResult(ConnectorConstants.INVALID_REQUEST_PAYLOAD,
                     String.format(ConnectorConstants.ERROR_PARAMETER_MISSING,
-                            ConnectorConstants.ACC_IDENTIFICATION));
+                            MT940Constants.DN_ACC_IDENTIFICATION));
         }
 
         if (account.length() > 35) {
```

Comment:
use the constant here

Reply:
Fixed in https://github.com/wso2-extensions/mediation-swift-iso20022-module/pull/6/commits/02976704c3798e54f6e5b27d539a212b8bdb900d